jwst/associations/lib/rules_level3_base.py

```diff
@@ -756,11 +756,25 @@ def __init__(self):
                 sources=['pupil', 'grating'],
                 required=False,
             ),
-            DMSAttrConstraint(
-                name='opt_elem3',
-                sources=['fxd_slit'],
-                required=False,
+
+            Constraint(
+                [
+                    DMSAttrConstraint(
+                        name='opt_elem3',
+                        sources=['fxd_slit'],
+                        value=['s200a1|s200a2'],
+                        force_unique=False,
+                        required=False,
+                    ),
+                    DMSAttrConstraint(
+                        name='opt_elem4',
+                        sources=['fxd_slit'],
+                        required=False,
+                    ),
```

Comment:
So let me make sure I understand the result of the new logic. The 2nd new `DMSAttrConstraint` will continue to create a level-3 asn for a set of exposures that contain any value of `FXD_SLIT`, while the 1st new constraint will make sure that sets of exposures that have `FXD_SLIT` values of both S200A1 and S200A2 end up in the same level-3 ASN. Right?

Reply:
Yes.